man/man8/zfs-hold.8

```diff
@@ -33,9 +32,10 @@
 .Dd June 30, 2019
 .Dt ZFS-HOLD 8
 .Os
+.
 .Sh NAME
 .Nm zfs-hold
-.Nd Hold a snapshot to prevent it being removed with the zfs destroy command.
+.Nd hold snapshot of ZFS dataset to prevent removal
```

Comment:
nit: I'd keep the "a" at least.

I'm not sure that "of ZFS dataset" is necessary.

How about "hold a snapshot to prevent its removal"

Reply:
Went with "hold ZFS snapshots to prevent their their removal"

Reply:
Sounds good (presuming that the double "their" was just a typo here)!